sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -1300,6 +1350,7 @@ class FileStreamSourceSuite extends FileStreamSourceTest {
           .option("checkpointLocation", checkpoint)
           .start(targetDir)
       }
+
```

Comment:
ditto. Please remove this extra empty line addition.

Reply:
fixed